### ISSUE TITLE
There is no keyboard navigation on the second and third custom menu. (CDTS application)

### ISSUE THREAD

**Issue by ahmad-shahid** [wet-boew/cdts-sgdc#695]
[Issue came thru email]

On the menu section, both second and third menu are link which is interactive elements. Keyboard user should be able to use tab key to navigate.
Reference: [Using ARIA menus - WCAG WG (w3.org)](https://www.w3.org/WAI/GL/wiki/Using_ARIA_menus#:~:text=Use%20TAB%20key%20to%20move%20focus%20to%20menu,bar%20loses%20focus%20and%20triggers%20an%20onblur%20event) and it fails on [WCAG2.1  2.4.3](https://www.w3.org/TR/WCAG21/#focus-order)

**Comment by ahmad-shahid** [comment]
Fixed thru GCWeb 19.3.1
